### DISCUSSION TITLE
Ayuda con las estadísticas

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#469] [Q&A]
Buenos días, ya logré sacar las estadísticas pero quiero ponerle como progress bar o algo similar.

**Comment by DanyVeneno** [comment]
Hola Gloria este es el repo  de otro compañero Explorer y creo que si trae un muy buen nivel  https://github.com/ErickMontesDK/Front-End-JS-Pokedex en lo personal tambien estoy atorado.
saludos buen dia

**Reply by GloriaCoralCerecedo** [reply]
Gracias la verdad que en mi pokedex no salio muy bien el diseño pero si funciona, solo quiero agregarle eso y otra cosita

**Reply by SujeiRodriguez** [reply]
yo soy fan de esa pokedex XD... le quedo con ganas, hasta se la pase a mi sobrinillo para que la usara

**Comment by chistoperez** [comment]
yo la hice con puro CSS, usé keyframes 
https://chistoperez.github.io/pokedex/

**Comment by micheldavrdz** [comment]
Si buscas una solución mas "nativa" HTML tiene una etiqueta llamada **meter**, tal vez sea lo que estas buscando (y utilizar CSS para ajustarla a como quieras que se vea)

[Link en MDN (Inglés)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meter)

**Comment by GabrielaEsquivel** [comment]
¡Hola! Yo lo he hecho con bootstrap y js te dejo aquí el enlace: 
https://getbootstrap.com/docs/4.0/components/progress/

mi ejemplo del progress en html: 
```
                     <div class="progress" style="height: 20px;">
                             <div class="progress-bar" role="progressbar" id="hpBar" style="width: 25%;" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100">
                            </div>
                   </div>
```

y mi JS:
```
 let barStat = document.getElementById(`${stats.stat.name}Bar`);
 barStat.style.width=`${stats.base_stat}%`
```
